### PR TITLE
Dept 13 logout button on every page

### PR DESCRIPTION
**Pull request by blamboy090**
Created Layout for application
Log out button persists on each page

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { UserProvider } from '@auth0/nextjs-auth0';
 import type { AppProps } from 'next/app';
 import { useEffect, useState } from 'react';
 
+import Layout from '@/components/layout/layout';
+import styles from '@/styles/Home.module.css';
```
Comment (agonxalez) [inline pages/_app.tsx:8]: Take this out and WE CALL IT GOOOOOOOOOOOOOOOOOOD

**Review by agonxalez — CHANGES_REQUESTED** [review]

File: pages/_app.tsx (hunk 1 above)
Comment (blamboy090) [reply pages/_app.tsx]: Updated!

**Review by agonxalez — APPROVED** [review]
lgtm

**Review by bdurel — APPROVED** [review]
Nice work Ben. This all looks good to me.

**Pull request merged**